packages/odata-service-inquirer/test/unit/prompts/cap-project/cap-helper.test.ts

```diff
@@ -138,6 +145,73 @@ describe('cap-helper', () => {
         expect(findCapProjectsSpy).toHaveBeenCalledWith({ 'wsFolders': ['/test/mock/'] });
     });
 
+    test('getCapProjectChoices (Windows paths)', async () => {
```

Comment:
test('Should use realpath to convert Windows paths to correct casing - getCapProjectChoices')